SetupGuide_Plugin_Admin.php

```diff
@@ -1277,6 +1372,19 @@ private function get_config() {
 						<input type="checkbox" id="lazyload-enable" value="1" /> <label for="lazyload-enable">' .
 						esc_html__( 'Lazy Load Images', 'w3-total-cache' ) . '</label></p>',
 				),
+				array( // Google Pagespeed Tool.
+					'headline' => __( 'Google PageSpeed', 'w3-total-cache' ),
+					'id'       => 'gps1',
+					'markup'   => '<p>' .
+						esc_html__(
+							'This tool can be used to analyze your website\'s homepage using the Google PageSpeed Insights API to gather desktop/mobile performance metrics. Additionally for each metric W3 Total Cache will include an explaination of the metric and our recommendation for achieving improvments via W3 Total Cache features/extensions if available.',
+							'w3-total-cache'
+						) . '</p><br/><p>' .
```

Comment:
The `<br/>` may not be needed.  It adds too much space between paragraphs.

Reply:
Removed.